server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -553,7 +565,24 @@ private ServletHolder getRestServlet() {
     }
   }
 
+  public static class CompactionStats {
+    public final long count;
+    public final Long oldest;
+    public final long fetched;
+
+    CompactionStats(List<ActiveCompaction> active) {
+      this.count = active.size();
+      long oldest = -1;
+      for (ActiveCompaction a : active) {
+        oldest = Math.max(oldest, a.age);
+      }
+      this.oldest = oldest < 0 ? null : oldest;
+      this.fetched = System.currentTimeMillis();
```

Comment:
This should use nano time to compute time durations.

Reply:
A lot of this code was copy and pasted but I can do a change replacing `System.currentTimeMillis()` in the Monitor.